tools/test-backend

```diff
@@ -260,8 +260,14 @@ def main() -> None:
                         default=False,
                         help=("Generate Stripe test fixtures by making requests to Stripe test network"))
     parser.add_argument('args', nargs='*')
+    parser.add_argument('--ban-extra-output', dest='ban_spam',
+                        action="store_true",
+                        default=False, help='Throw error on detecting extra spam output in stdout and stderr.')
 
     options = parser.parse_args()
+    if options.ban_spam:
+        os.environ["BAN_TEST_OUTPUT_EXTRA_SPAM"] = "1"
```

Comment:
I'd encourage using the same name for a variable like this, rather than having 3 versions:
* BAN_TEST_OUTPUT_EXTRA_SPAM, ban-extra-output, ban_spam.

Reply:
Maybe the name should be `--ban-console-output` as the name and use that consistently (there should be no references to "spam" to "output_extra" in this PR)